operations/deployment/crunchy-postgres/templates/networking/portForward.yaml

```diff
@@ -0,0 +1,33 @@
+{{ if .Values.userInterface.enabled }}
+
+# Port forwarding so we can access pgAdmin console if enabled
+# https://access.crunchydata.com/documentation/postgres-operator/v5/architecture/pgadmin4/
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: "{{ .Release.Name }}"
+  labels:
+    app.kubernetes.io/managed-by: {{ .Release.Service | quote }}
+    app.kubernetes.io/instance: {{ .Release.Name | quote }}
+    app.kubernetes.io/version: {{ .Chart.AppVersion }}
+    helm.sh/chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+  annotations:
+    "helm.sh/hook": post-install
+    "helm.sh/hook-weight": "-5"
+    "helm.sh/hook-delete-policy": hook-succeeded
+spec:
+  template:
+    metadata:
+      name: "{{ .Release.Name }}"
+      labels:
+        app.kubernetes.io/managed-by: {{ .Release.Service | quote }}
+        app.kubernetes.io/instance: {{ .Release.Name | quote }}
+        helm.sh/chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    spec:
+      restartPolicy: Never
+      containers:
+      - name: post-install-job
+        image: "alpine:3.3"
+        command: kubectl port-forward svc/{{ .Release.Name }}-pgadmin 5050:5050
```

Comment:
should we let the user define the ports? Or is 5050 a pgAdmin requirement?

Reply:
https://access.crunchydata.com/documentation/postgres-operator/v5/architecture/pgadmin4/

According to this I believe it's required to be port 5050.